bindings/nodejs/src/lib.rs

```diff
@@ -642,17 +642,17 @@ impl Operator {
     /// ### Example
     ///
     /// ```javascript
-    /// const req = await op.presignRead(path, parseInt(expires));
+    /// const req = await op.presignRead(path, BigInt(expires));
     ///
     /// console.log("method: ", req.method);
     /// console.log("url: ", req.url);
     /// console.log("headers: ", req.headers);
     /// ```
     #[napi]
-    pub async fn presign_read(&self, path: String, expires: u32) -> Result<PresignedRequest> {
+    pub async fn presign_read(&self, path: String, expires: BigInt) -> Result<PresignedRequest> {
```

Comment:
I don't think we really need `BigInt`, it usually less than one day (24 * 3600).